modules/ROOT/examples/live-demos/revisionhistory/index.js

```diff
@@ -183,17 +183,17 @@ const revisions = [
 ];
 
 const revisionhistory_fetch_revision = (_editor, revision) =>
-  new Promise((resolve) => {
+  new Promise((resolve, reject) => {
     setTimeout(() => {
-      let newRevision = null;
       for (let i = 0; i < revisions.length; i++) {
         const temp = revisions[i];
         if (temp.revisionId === revision.revisionId) {
-          newRevision = temp;
+          resolve(temp);
           break;
         }
       }
-      resolve(newRevision);
+      // reject has no effect when the promise is resolved
```

Comment:
I appreciate that this is how promises work, but I would still like to encourage better coding practices - particularly for our code examples we expect customers to use (and often deploy to production).

That means either using an `else` block or switching to an `async` function and early `return`.